collector/filesystem_netbsd.go

```diff
@@ -0,0 +1,132 @@
+// Copyright 2024 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build !nofilesystem
+// +build !nofilesystem
+
+package collector
+
+import (
+	"fmt"
+	"github.com/go-kit/log/level"
```

Comment:
This was dropped in https://github.com/prometheus/node_exporter/commit/090957658ee7be8c7c71b2724979b36c49371b20 and re-introduced here without syncing dependencies, hence the issue on `master`. Re-dropped in [#3131](https://github.com/prometheus/node_exporter/pull/3131/files#diff-2d11ce76766900464657fa926b3cb617390db5e39936de9baa7ed902101b150dL21).